sensors/nxt_i2c_sensor_defs.c

```diff
@@ -262,6 +262,19 @@ static int ht_angle_scale(void *context,
 	return 0;
 }
 
+static int ms_pps58_nx_set_mode_post_cb(nxt_i2c_sensor_data *sensor, u8 mode)
+{
+	struct i2c_client *client = sensor->client;
+	int ref;
+
+	if(mode < 4 || 6 < mode)
+		return 0;
+
+	ref = i2c_smbus_read_word_data(client, 0x43);
+	if(ref < 0)
+		return ref;
+	return i2c_smbus_write_word_data(client, 0x47, ref);
```

Comment:
This is a mode callback, so will not change the behavior of the ZERO command at all. 

If you want a precise relative pressure reading, then yes, you should use the ZERO command to tare the value. But my thinking here is that we use standard atmospheric pressure here to get you in the ballpark so that you don't *have* to zero the sensor if you don't want to.